app/helpers/dashboard_helper.rb

```diff
@@ -41,4 +41,33 @@ def any_successes?(statuses)
   def any_failures?(statuses)
     (%i[invalid not_marc] & statuses).any?
   end
+
+  def dump_record_count(dump)
+    0 if dump.nil?
+
+    if dump.marcxml&.attachment && dump.marcxml.attachment.metadata
+      # Use marcxml as the source of the record count. marc21 count may differ.
+      dump.marcxml.attachment.metadata['count']
+    else
+      0
+    end
```

Comment:
I'd love to see this as a method on `NormalizedDump`; I can see using it some other places (like in the OAI controller) too.